desktop/src/renderer/src/App.tsx

```diff
@@ -55,9 +55,9 @@ function App(): JSX.Element {
 
 	return (
 		<BrowserRouter>
-			<div style={{ display: "flex" }}>
+			<div style={{ display: "flex", width: "100%" }}>
 				<SideNav accessToken={accessToken} idToken={idToken} />
-				<main style={{ flexGrow: 1 }}>
+				<main>
```

Comment:
I think this should be left as it is in main

Reply:
If there were 2 components that had flex grow set we could change what ratio they flex in relation to each other, as it is right now this doesnt seem to change anything.